opencensus/stats/internal/tag_set.cc

```diff
@@ -61,5 +63,17 @@ bool TagSet::operator==(const TagSet& other) const {
   return tags_ == other.tags_;
 }
 
+std::string TagSet::DebugString() const {
```

Comment:
Add a comment that the format is not guaranteed so people cannot depend on this to retrieve tags.

Reply:
That comment is already in the header. Should I repeat it here?